### ISSUE TITLE
Enhance accessibility

### ISSUE THREAD

**Issue by bpcurse** [bpcurse/nextcloud-userexport#11]
Enhance accessibility for disabled users through html design and labelling.

**Comment by bpcurse** [comment]
Can't tell how well this script performs regarding accessibility. Will try to keep it in mind for further code additions. If there are issues please open a new issue describing the point of failure.
